### DISCUSSION TITLE
Failed to load resource

### DISCUSSION THREAD

**Discussion by Eaclean** [Leantime/leantime#1446] [Q&A] [answered]
Hi,

Was wondering if anyone has ever seen this error in dev tools?

I'm on an IIS instance.

Thanks

![Capture](https://user-images.githubusercontent.com/123482922/219675950-d816511f-9d67-46f9-b01b-9ef85b50bd7a.PNG)

**Comment by marcelfolaron** [comment]
Thats good odd.
Anything in logs/error.log ?

**Reply by Eaclean** [reply]
Nothing apart from a deprecated warning.

[17-Feb-2023 14:09:00 Europe/London] PHP Deprecated:  Automatic conversion of false to array is deprecated in E:\leantime\app\domain\menu\repositories\class.menu.php on line 176

**Reply by Eaclean** [reply]
This one can also appear.

![image](https://user-images.githubusercontent.com/123482922/219679446-726674ee-4935-434f-b014-b83a5ce7d5d6.png)

**Comment by marcelfolaron** [comment]
Can you set $debug to 1 in your config file and check the error log again?
(Browser cache refresh required)

**Reply by Eaclean** [reply]
Yea currently got debug on but no errors apart from that deprecated one.

Thanks

**Comment by marcelfolaron** [comment]
Can you open /api/session in your browser and see if something shows up.

**Reply by Eaclean** [reply]
Just get a 404 for that. Do you get anything?

Thanks

**Comment by marcelfolaron** [comment]
Oh sorry, should be /api/sessions (with the s at the end)

**Reply by Eaclean** [reply]
Just get a blank page for that, no 404.

**Comment by Eaclean** [comment, marked as answer]
I noticed that Leantime seems to send some API requests via the PATCH method (rather than GET, or POST).  

I’ve updated the handler mappings for PHP 8.1.4 to include extended verbs for DEBUG, DELETE, PUT, and PATCH:
